packages/core/content-releases/server/src/bootstrap.ts

```diff
@@ -69,6 +69,10 @@ export const bootstrap = async ({ strapi }: { strapi: LoadedStrapi }) => {
 
           throw err;
         });
+
+      Object.entries(ALLOWED_WEBHOOK_EVENTS).forEach(([key, value]) => {
+        strapi.webhookStore.addAllowedEvent(key, value);
+      });
```

Comment:
I see how this makes sense and makes it easy to update in the future if we add more events. But I dunno maybe that's a future problem and we should simply do:

`strapi.webhookStore.addAllowedEvent('RELEASES_PUBLISH', 'releases.publish');`

Up to you I'm fine with what you have too

Reply:
ahh I see you are using the constant elsewhere, my bad. You can disregard.